test/dshell/extra-files/dll/testmodule.d

```diff
@@ -0,0 +1,10 @@
+module run.unicode_06_哪里;
+
+int 哪里(int ö){
+        return ö+2;
+}
+
+version(Windows) {
+    import std.algorithm : canFind;
+    static assert(!哪里.mangleof.canFind("哪里"));
```

Comment:
why not test the full mangle instead ?

Reply:
The rest of the mangle is irrelevant to the test, which would mean that unrelated parts to it if changed will cause this to error. This isn't ideal, but neither is the phobos import.